Description: another path that kills cmd invocations
 Nothing new to document
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Bug-Debian: https://bugs.debian.org/776821
Bug-Debian: https://bugs.debian.org/804026
Bug-Debian: https://bugs.debian.org/813338

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2016-07-31

--- icu-57.1.orig/source/config/mh-mingw64
+++ icu-57.1/source/config/mh-mingw64
@@ -95,10 +95,10 @@ LIBICU = $(STATIC_PREFIX_WHEN_USED)$(ICU
 # The #M# is used to delete lines for icu-config
 # Current full path directory.
 #CURR_FULL_DIR=$(shell pwd -W)#M# for MSYS
-CURR_FULL_DIR?=$(subst \,/,$(shell cmd /c cd))#M# for Cygwin shell
+CURR_FULL_DIR?=$(subst \,/,$(shell pwd))#M# for Cygwin shell
 # Current full path directory for use in source code in a -D compiler option.
 #CURR_SRCCODE_FULL_DIR=$(subst /,\\\\,$(shell pwd -W))#M# for MSYS
-CURR_SRCCODE_FULL_DIR=$(subst \,/,$(shell cmd /c cd))#M# for Cygwin shell
+CURR_SRCCODE_FULL_DIR=$(subst \,/,$(shell pwd))#M# for Cygwin shell
 
 ## Compilation rules
 %.$(STATIC_O): $(srcdir)/%.c
